src/Tasks.UnitTests/Copy_Tests.cs

```diff
@@ -126,10 +132,7 @@ public void CopyWithNoInput()
         {
             var task = new Copy { BuildEngine = new MockEngine(true), };
             task.Execute().ShouldBeTrue();
-            task.CopiedFiles.ShouldNotBeNull();
-            task.CopiedFiles.Length.ShouldBe(0);
-            task.DestinationFiles.ShouldNotBeNull();
-            task.DestinationFiles.Length.ShouldBe(0);
+            (task.CopiedFiles == null || task.CopiedFiles.Length == 0).ShouldBeTrue();
```

Comment:
nit: Destination files are not checked anymore in this test. Is that intentional?